azuredeploy-large-ha.json

```diff
@@ -7,7 +7,7 @@
             "metadata": {
                 "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
             },
-            "defaultValue": "https://raw.githubusercontent.com/Azure/Moodle/master/"
+            "defaultValue": "https://raw.githubusercontent.com/migel0/Moodle/edit/appgwfileshare/"
```

Comment:
This moves the authoratative repo to your own person repo. This cannot be changed. Furthermore the URL is a 400 and thus will not work.